src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -238,32 +237,35 @@ async def set_output_lut(self, lut_index, output):
             :py:meth:`set_tpz_io_settings<Tpz.set_tpz_io_settings>`
             method.
         """
+        if self.voltage_limit is None:
+            raise ValueError("Voltage limit is not set")
+
         volt = round(output * 32767 / self.voltage_limit)
-        payload = st.pack("<HHH", 1, lut_index, volt)
+        payload = st.pack("<HHH", _CHANNEL, lut_index, volt)
         await self.send(Message(MGMSG.PZ_SET_OUTPUTLUT, data=payload))
 
-    async def get_output_lut(self):
+    async def get_output_lut(self) -> tuple[int, float]:
         """Get the ouput LUT values for WGM (Waveform Generator Mode).
 
         :return: a tuple whose first element is the lut index and the second is
             the voltage output value.
         :rtype: a 2 elements tuple (int, float)
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_OUTPUTLUT, [MGMSG.PZ_GET_OUTPUTLUT], 1
+            MGMSG.PZ_REQ_OUTPUTLUT, [MGMSG.PZ_GET_OUTPUTLUT], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_OUTPUTLUT, [MGMSG.PZ_GET_OUTPUTLUT], Tpz._CHANNEL
```
Use class constants.